targets/UnrealMarketplacePlugin/examplesource/ExampleProject/BuildAndroidOnLinux.sh.ejs

```diff
@@ -0,0 +1,40 @@
+!/bin/bash
+
+# echo ========== BUILDING ANDROID EXAMPLE PROJECT ==========
+usage="USAGE: BuildAndroidOnLinux.sh [archive directory]"
+
+if "%~1"=="" then USAGE()
+
+# Unreal Path, to change depending on your workspace setup.
+uePath=/Users/Shared/Epic Games
+
+# Unreal Version, to change on every unreal version
+ueVersion=<%- ueTargetVersion %>
```

Comment:
are we generating one script per ueVersion? Any reason not to simply expose this as a script parameter?